code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,10 +15,13 @@
 		insert(holder)
 	..()
 
+
 /obj/item/organ/internal/proc/insert(mob/living/carbon/M, special = 0)
 	if(!iscarbon(M) || owner == M)
 		return
 
+	processing_objects -= src
```

Comment:
well tosse me on what i need to add where...i will get to it later. got to sleep soonish